python-regression/IXI/export-spent/index.js

```diff
@@ -0,0 +1,79 @@
+var System = java.lang.System;
+
+var spentAddressProvider = IOTA.spentAddressesProvider
+
+var iri = com.iota.iri;
+var Callable = iri.service.CallableRequest;
+var Response = iri.service.dto.IXIResponse;
+var ErrorResponse = iri.service.dto.ErrorResponse;
+
+var fileName = "spentAddresses.txt";
+
+// Log using logger of the ixi class
+var log = org.slf4j.LoggerFactory.getLogger(iri.IXI.class);
+
+/*
+curl http://localhost:14265 -X POST -H 'X-IOTA-API-Version: 1.4.1' -H 'Content-Type: application/json' -d '{"command": "Spent.generateSpentAddressesFile"}'
+*/
+function generate(request) {
+    var file = new java.io.File(fileName);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['java' is not defined. (no-undef)](https://app.codacy.com/manual/GalRogozinski/iri/pullRequest?prid=3955655)